protected-api-access-protected-api/app.js

```diff
@@ -64,27 +64,29 @@ const validateJwt = (req, res, next) => {
       issuer: config.auth.authority + '/v2.0'
     }
 
+    // Get Signing Keys
+    const getSigningKeys = (header, callback) => {
+      jwksClient.getSigningKey(header.kid, function (err, key) {
+        const signingKey = key.publicKey
+        callback(null, signingKey)
+      })
+    }
```

Comment:
Is it possible to convert it to look like this, or does that not work well in the callback invocation?

```suggestion
    // Look up the public key used to sign JWT tokens, as provided by the OIDC discovery endpoint
    const getSigningKeys = (header, callback) => {
      jwksClient.getSigningKey(header.kid, function (err, key) {
        callback(null, key.publicKey)
      })
    }
```

Reply:
Will give it a shot

Reply:
Oh, yeah, that works. that's actually what I have on my latest version

Reply:
thx!